sdk/template/azure-template/inc/azure/template/version.hpp

```diff
@@ -18,12 +18,12 @@ namespace Azure { namespace Template {
     static constexpr int Minor = AZURE_TEMPLATE_VERSION_MINOR;
     static constexpr int Patch = AZURE_TEMPLATE_VERSION_PATCH;
     static std::string const PreRelease;
-    static std::string const VersionString();
+    static std::string VersionString();
 
   private:
-    //To avoid leaking out the #define values we smuggle out the value
+    // To avoid leaking out the #define values we smuggle out the value
     // which will later be used to initialize the PreRelease std::string
-    static constexpr const char* secret = AZURE_TEMPLATE_VERSION_PRERELEASE;
```

Comment:
Keep this as secret b/c the value is special and only exists to allow us to smuggle the data out

Reply:
I don't understand what we're achieving by naming it "cryptically", not following the naming guidelines, since it is already private, but I don't have that strong of an opinion as you have, and I don't think it is that important - it's just a name, and everything is pretty much is clear as a day, no matter how you name it.